src/shared/entity.model.ts

```diff
@@ -8,6 +8,11 @@ import {
 } from 'typeorm';
 import { v4 as uuid } from 'uuid';
 
+// TODO: not sure if there is a way to add this
+// using annotation or static class method
+// to be shared with all Model extending the `Model`
+export const POSTS_SCHEMA = 'pickify_posts';
```

Comment:
Regarding the comment.
This could be done, we can add `POSTS_SCHEMA` as a global injectable/provider. [See documentation of non-class-based provider](https://docs.nestjs.com/fundamentals/custom-providers#non-class-based-provider-tokens)
I think this is the better way, or the nestjs way, of doing things, but I'm not really sure if it is worth the effort.
Please let me know if it is important to do, or at least the better way.

Reply:
will the way you suggest work well with the migration scripts?

Reply:
No, migration scripts aren't loaded into nestjs during runtime.

Reply:
that's why I did it this way. I am using the same constant in the migration script.